cypress/support/pages/home.pageObject.js

```diff
@@ -8,8 +8,15 @@ class HomePageObject extends PageObject {
   }
 
   assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+    this.usernameLink.should('contain', username);
+  }
+
+  clickNewArticle() {
+    cy.get('a.nav-link').contains('New Article').click();
+  }
+
+  clickSettings() {
+    cy.get('a.nav-link').contains('Settings').click();
   }
 }
```

Comment:
The file name 'home.pageObject.js' does not match the default export 'HomePageObject'. Consider renaming either the file or the class for consistency.

Reply:
I have the same situation for all pages in the project and can't understand the problem of this approach.